src/generator.js

```diff
@@ -17,7 +17,13 @@ export function bodyFromSchema(schema, payload, parser, contentType = 'applicati
   let asset = null;
 
   try {
-    let body = schema.example || faker(schema);
+    let body;
+
+    if (schema.examples && schema.examples[0]) {
+      body = schema.examples[0];
+    } else {
+      body = faker(schema);
```

Comment:
faker respects `schema.example`?

Reply:
No, `example` will never get here because that isn't valid in JSON Schema which is being passed in. This line will actually go away in https://github.com/apiaryio/fury-adapter-swagger/pull/203/commits/abed7a4a44d396a8ce6e6790138b2094c262f6ea once JSON Schema Faker is released.